plugins/Highway_Lanes.py

```diff
@@ -111,14 +111,17 @@ def way(self, data, tags, nds):
                     if not unknown:
                         # merge_to_left is a on the right and vice versa
                         t = tags_lanes[tl] \
-                            .replace("left", "l").replace("slight_left", "l").replace("sharp_left", "l") \
+                            .replace("slight_left", "l").replace("sharp_left", "l") \
                             .replace("through", " ") \
-                            .replace("right", "r").replace("slight_right", "r").replace("sharp_right", "r") \
+                            .replace("slight_right", "r").replace("sharp_right", "r") \
                             .replace("reverse", "U") \
                             .replace("merge_to_left", "r").replace("merge_to_right", "l") \
-                            .replace("none", " ").replace(";", "").split("|")
-                        t = ''.join(map(lambda e: " " if len(e) == 0 or e[0] != e[-1] else e[0], map(sorted, t)))
+                            .replace("left", "l").replace("right", "r") \
+                            .replace("none", "N").replace(";", "").split("|")
+                        t = ''.join(map(lambda e: "N" if len(e) == 0 else " " if e[0] != e[-1] else e[0], map(sorted, t)))
                         t = t.replace('U', '') # Ignore reverse
+                        # Ignore single none on the outside lanes: it could be a bus lane
+                        t = t.replace('rN', 'r').replace('Nl', 'l').replace('N', ' ')
```

Comment:
Should we not ensure N is first/last/only lane to remove it ? What about multiple `N` - Should we replace `N` at all ?

Reply:
I've updated the PR to ensure it only replaces the outmost lanes.
I believe that it is very unlikely that a dedicated bus lane would be in between lanes, or that there would be multiple dedicated bus lanes on a single road. However, the check would lose a lot of its strength if all 'nones' would become unchecked, as it very often refers to "regular" lanes being "through"-lanes without road marking.
Hence I think it's better to keep treating `N` as `through` in the majority of cases, i.e. `left|||right|` very likely means `left|no turn = through|no-turn=through|right|some-special-lane`